1.0.0-beta4/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mono:3.12
```

Comment:
@vlesierse do you think we need to use mono as base in this case where we're compiling/installing mono from source? in any case we should probably upgrade to `mono:4` here.